ide/projectui/src/org/netbeans/modules/project/ui/ProjectChooserAccessory.java

```diff
@@ -342,6 +359,7 @@ private void setAccessoryEnablement( boolean enable, int numberOfProjects ) {
         jTextFieldProjectName.setEnabled( enable );
         jTextFieldProjectName.setForeground(/* i.e. L&F default */null);
         jCheckBoxSubprojects.setEnabled( enable );
+        jCheckBoxPrime.setEnabled(enable);
```

Comment:
I'd enable iff any of the selected project(s) provides such an action; e.g. Ant projects do not.

Reply:
OK: commands.get(commandId)